articles/azure-app-configuration/howto-best-practices.md

```diff
@@ -102,6 +102,14 @@ When you use App Configuration in client applications, ensure that you consider
 
 To address these concerns, we recommend that you use a proxy service between your client applications and your App Configuration store. The proxy service can securely authenticate with your App Configuration store without a security issue of leaking authentication information. You can build a proxy service by using one of the App Configuration provider libraries, so you can take advantage of built-in caching and refresh capabilities for optimizing the volume of requests sent to App Configuration. For more information about using App Configuration providers, see articles in Quickstarts and Tutorials. The proxy service serves the configuration from its cache to your client applications, and you avoid the two potential issues that are discussed in this section.
 
+## Configuration as Code
+Configuration as code is a practice of managing configuration files under your source control system, for example, a git repository. It brings you the benefits like traceability and approval process for any configuration changes. If you adopt configuration as code practice, App Configuration has tools to assist you in deploying your configuration data. This way, your applications can access the latest data from your App Configuration store(s). 
```

Comment:
More benefits are also added security and standardization. 
Nits: Configuration as code is the practice of managing the configuration files in your source control system ...
brings you the benefits  -> gives you benefits
practice -> practices

Reply:
Updated. Thanks.

There should be a doc explaining what CaC is. Unfortunately, I couldn't find one in Azure docs. There are many articles on the internet. We could write one in the future. This doc, however, is not meant to explain what CaC is, so it's by no means an exhaustive list of benefits. Some benefits depend on how you implement CaC with your team, so I don't want to get into too much detail about benefits without discussing what practices to take. We are not trying to sell CaC. We simply offer solutions to customers who have already chosen CaC for whatever reasons.